guideline-14.md

```diff
@@ -1,5 +1,7 @@
 <h4>14. Frequent commits to a plugin should be avoided.</h4>
 
-The SVN repository is a release repository, not a development one. All commits, code or readme files, will trigger a regeneration of the zip files associated with the plugin, so only code that is ready for deployment (be that a stable release, beta, or RC) should be pushed to SVN. Including a descriptive and informative message with each commit is strongly recommended. Frequent ‘trash’ commit messages like ‘update’ or ‘cleanup’ makes it hard for others to follow changes. Multiple, rapid-fire commits that only tweak minor aspects of the plugin (including the readme) cause undue strain on the system and can be seen as gaming Recently Updated lists.
+The SVN repository is a release repository, not a development one. All commits, code or readme files, will trigger a regeneration of the zip files associated with the plugin, so only code that is ready for deployment (be that a stable release, beta, or RC) should be pushed to SVN.
```

Comment:
Not specific to your changes, but should “SVN” be changed to “Subversion” or include an acronym tag?

Same with “RC”. I’d write that as “be that an alpha, beta, release candidate, or stable release” to maintain the order of progression.